### PR TITLE
PE-2466: feat: add bar tags

### PR DESCRIPTION
**Pull request by jdaev**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/6jrmui366cu8g
iOS release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:ios:06ea33a95a2e0d42ffce07/releases/22cg081vkk5ro

### CODE REVIEW

**Comment by arielmelendez** [comment]
mah man

File: lib/entities/entity.dart [hunk 1]
```diff
@@ -109,4 +109,13 @@ extension TransactionUtils on TransactionBase {
   void addArFsTag() {
     addTag(EntityTag.arFs, '0.11');
   }
+
+  void addBarTags() {
+    addTag(EntityTag.protocolName, 'BAR');
+    addTag(EntityTag.action, 'Burn');
+    addTag(EntityTag.appName, 'SmartWeaveAction');
+    addTag(EntityTag.appVersion, '0.3.0');
```
Comment (matibat) [inline lib/entities/entity.dart:117]: We'll have duplicate tags. I don't think it collides with ArDrive's logic, but maybe with the contract... (?

File: lib/blocs/upload/upload_handles/file_v2_upload_handle.dart [hunk 2]
```diff
@@ -66,7 +66,8 @@ class FileV2UploadHandle implements UploadHandle {
           : Transaction.withBlobData(data: fileData),
       wallet,
     )
-      ..addApplicationTags(version: version);
+      ..addApplicationTags(version: version)
+      ..addBarTags();
```
Comment (matibat) [inline lib/blocs/upload/upload_handles/file_v2_upload_handle.dart:70]: Would this BURN all v2 uploads?

Comment (matibat) [reply lib/blocs/upload/upload_handles/file_v2_upload_handle.dart:70]: Only v2? Are bundles excluded?

**Review by matibat — COMMENTED** [review]
I'm not 100% sure about my comments. Please consider those as questions only.

**Review by thiagocarvalhodev — APPROVED** [review]
LGTM!

**Pull request merged**